README.md

```diff
@@ -73,22 +74,20 @@ Latest releases:
 * [3.1.1](https://esl.github.io/MongooseDocs/3.1.1/)
 * [3.0.1](https://esl.github.io/MongooseDocs/3.0.1/)
 
-
 **MongooseIM documentation highlights:**
 
 When developing new features/modules, please make sure you add basic documentation to the 'doc/' directory, and add a link to your document in 'doc/README.md.'
 
 * [Tutorials](https://esl.github.io/MongooseDocs/latest/user-guide/How-to-build/). Learn how to:
-    * [Build MongooseIM from source code](doc/user-guide/How-to-build.md)
-    * [Set up MongoosePush](doc/user-guide/push-notifications/Push-notifications.md)
-    * [Set up MongooseICE](doc/user-guide/ICE_tutorial.md)
-    * [Build an iOS messaging app](doc/user-guide/iOS_tutorial.md)
-* [User Guide](https://esl.github.io/MongooseDocs/latest/user-guide/ABCs-of-MongooseIM/). Learn all about how to use MongooseIM in your project. Explore its features, supported XEPs, RFCs and database backends, as well as its architecture and deployment strategies.
-* [Configuration](https://esl.github.io/MongooseDocs/latest/Basic-configuration/). Explore some of the available options including database backend configuration, access control lists, listener and extension modules.
+    * [Build MongooseIM from source code](https://esl.github.io/MongooseDocs/latest/user-guide/How-to-build.md)
+    * [Set up MongoosePush](https://esl.github.io/MongooseDocs/latest/user-guide/push-notifications/Push-notifications.md)
+    * [Set up MongooseICE](https://esl.github.io/MongooseDocs/latest/user-guide/ICE_tutorial.md)
+    * [Build an iOS messaging app](https://esl.github.io/MongooseDocs/latest/user-guide/iOS_tutorial.md)
+* [User Guide](https://esl.github.io/MongooseDocs/latest/user-guide/Features/). Learn all about how to use MongooseIM in your project. Explore its features, supported XEPs, RFCs and database backends, as well as its architecture and deployment strategies.
+* [Configuration](https://esl.github.io/MongooseDocs/latest/advanced-configuration/configuration-files/). Explore available options including database backend configuration, access control lists, listener and extension modules.
```

Comment:
Yes, `advanced` should be dropped - good catch!
I've changed some of the links to be absolute, because:
1. The section is "documentation highlights", so for me it seems it should lead to our hosted documentation.
2. Some of the links were already absolute, so it was really unifying them one way or the other.